src/data/mote/moter_hooks.ts

```diff
@@ -1,7 +1,7 @@
 import { useAppSelector } from "../../hooks/hooks";
 import { MoteDTO } from "./types/moteTypes";
 
```

Comment:
Kom på at jeg har lagt andre hooks som henter fra state i src/hooks, men mulig de burde flyttes inn i riktig data-subfolder slik som denne 🤔 Hva tenker du? Ikke noe som må gjøres i denne PRen, altså.

Reply:
Litt usikker på hva som blir best her. Hooks går jo ofte på tvers av reducere og domener, og da blir en domene-oppdeling av redux litt rart synes jeg. For min del er jeg mer fan av å strukturere redux i actions/reducers/hooks/sagas. Men ser at her har man brukt domene-strukturering. Dersom vi skal fortsette med domene-struktur bør vi sannsynligvis ha en måte å ha ting som går på tvers, f.eks "common" (eller bare /hooks som du har laget) eller noe.